stdlib/private/SwiftPrivate/IO.swift

```diff
@@ -13,6 +13,71 @@
 import Swift
 import SwiftShims
 
+#if os(Windows)
+import MSVCRT
+import WinSDK
+#endif
+
+#if os(Windows)
+public struct _FDInputStream {
+  public var handle: HANDLE = INVALID_HANDLE_VALUE
+  public var isEOF: Bool = false
+  public var isClose: Bool { return handle == INVALID_HANDLE_VALUE }
```

Comment:
[Should be `isClosed`](https://github.com/apple/swift/pull/21495/files#diff-743a80ce764d062f9fea22c775bc27a5R83).